.github/workflows/redis-suite.yml

```diff
@@ -4,21 +4,51 @@ on:
   schedule:
     - cron: '0 0 * * *'
   workflow_dispatch:
+    inputs:
+      redis_repo:
+        description: 'redis repo owner and name'
+        required: false
+        default: 'redis/redis'
+      redis_branch:
+        description: 'redis git branch or sha to use'
+        required: false
+        default: 'unstable'
+      redisraft_repo:
+        description: 'redisraft repo owner and name'
+        required: false
+        default: 'redislabs/redisraft'
+      redisraft_branch:
+        description: 'redisraft git branch or sha to use'
+        required: false
+        default: 'master'
 
 jobs:
   redis-suite:
     runs-on: ubuntu-latest
+    if: |
+      github.event_name == 'workflow_dispatch' ||
+      (github.event_name == 'schedule' && github.repository == 'redislabs/redisraft')
     steps:
+    - name: prep
+      if: github.event_name == 'workflow_dispatch'
+      run: |
+        echo "REDIS_REPOSITORY=${{github.event.inputs.redis_repo}}" >> $GITHUB_ENV
+        echo "REDIS_BRANCH=${{github.event.inputs.redis_branch}}" >> $GITHUB_ENV
+        echo "REDISRAFT_REPOSITORY=${{github.event.inputs.redisraft_repo}}" >> $GITHUB_ENV
+        echo "REDISRAFT_BRANCH=${{github.event.inputs.redisraft_branch}}" >> $GITHUB_ENV
     - uses: actions/checkout@v2
+      with:
+        repository: ${{ env.REDISRAFT_REPOSITORY }}
```

Comment:
@tezc Did you test this as a scheduled workflow? I'd expect these variables have to be defined (to hard coded value) in that case, but maybe I'm missing something.

Reply:
@yossigo You're right, I didn't try with scheduled workflow. It should be fixed right now.